lib/energy_globe_awards/reprocess.rb

```diff
@@ -0,0 +1,81 @@
+require 'json'
+require 'text/hyphen'
+
+
+class Reprocess
+  VALID_CATEGORIES = %w(earth air water fire youth other)
+
+  def initialize source, destination
+    @source      = source
+    @destination = destination
+    @json        = nil
```

Comment:
Oh, one tiny detail: When you call any method before calling `load_file`, it will give you an `undefined method bla for NilClass` or whatever. This might be intended, but you could get around it by assigning `[]` instead of `nil` here.